examples/sssp/sssp.cu

```diff
@@ -47,21 +43,16 @@ void test_sssp(int num_arguments, char** argument_array) {
   thrust::device_vector<vertex_t> predecessors(n_vertices);
```

Comment:
If you get a chance, can you test `thrust:device_vector` performance vs. `cudaMalloc/cudaFree` within SSSP? If passed in as `.data()` it is a synchronized call, so it shouldn't really be any better or worse, but I need some confirmation of that---I tried this a while ago, so just need another sanity check.